modules/regular_service_perimeter/main.tf

```diff
@@ -28,13 +28,15 @@ resource "google_access_context_manager_service_perimeter" "regular_service_peri
 
   status {
     restricted_services = var.restricted_services
+    resources           = [for item in var.resources : can(regex("global/networks", item)) ? format("//compute.googleapis.com/%s", item) : format("projects/%s", item)]
```

Comment:
I think this is already adding the resource i.e. the project to the perimeter and then `google_access_context_manager_service_perimeter_resource` tries to add it again leading to:

```
STDERR: Error: Unable to create ServicePerimeterResource, existing object already found: map[resource:projects/838237599229]

  with module.example.module.regular_service_perimeter_1.google_access_context_manager_service_perimeter_resource.service_perimeter_resource["838237599229"],
  on ../../../modules/regular_service_perimeter/main.tf line 229, in resource "google_access_context_manager_service_perimeter_resource" "service_perimeter_resource":
 229: resource "google_access_context_manager_service_perimeter_resource" "service_perimeter_resource" {
---- End output of terraform apply -auto-approve -lock=true -lock-timeout=0s -input=false -no-color -paral
```

Reply:
@cmalpe can u plz remove resource = ....

Reply:
This change is done @imrannayer